backend/app/project/routes.py

```diff
@@ -1,8 +1,57 @@
-import uuid
 from flask import jsonify, request, abort
 from app import db
-from app.models import Project
 from . import project
+from app.models import Project
+
+
+# get all endpoints
+@project.route("", methods=["GET"])
+def get_projects():
+    projects = Project.query.all()
+    return jsonify(Project.serialize_list(projects))
+
+
+# create a new project
+@project.route("", methods=["POST"])
```

Comment:
```suggestion
@project.route("/create-project", methods=["POST"])

Reply:
Yes sorry again, should not have pulled from a while back, please ignore this branch for now. I'm hoping that we get the "get_project" endpoint merged in and this branch should be trivial after that.

Reply:
I have not changed this yet because according to the articles I have attached below, the best practice is to not expose CRUD function names. Let me know if you decide against this Im more than happy to quickly amend it. 

Refer to "Never use CRUD function names in URIs"
https://stoplight.io/blog/crud-api-design/

The article also raises questions to the update endpoint, as `@project.route("/<uuid:id>/update", methods=["PUT"])` directly exposes the update action.

Reply:
I'm ok with just `""` as that's partly why I suggested `""` for #98 . @celinehuang If you're ok with it, @peterghrong do you mind changing it for the update route as well?

Reply:
Sure thing